### ISSUE TITLE
Rethink heading structure

### ISSUE THREAD

**Issue by jdalsem** [Elgg/Elgg#14435]
Currently we have inconsistency in the heading usage on Elgg site. We need to rethink how they are used.

Currently we roughly have this approach:
H1: site title
H2: page title (like 'All blogs' or 'blog title'
H3: entities summary titles (on 'All blog')

But sometimes we also are missing a h2 (like on Home) or have duplicate h3 elements like entity listings in widgets (widget title + entity summary title)

On detail pages with editor content (like blog/view) we have duplicate h2 elements (blog title and content h2)

After a quick brainstorm we came up with a new approach:

H1: should always be the page title (like 'All blogs') and not the site name / logo. Also on pages without a page title there should be at least a hidden page title

H2: item listing titles or widget titles, sidebar elements and content headings (made with ckeditor), module titles

H3: item listings when using in widgets or in content listings within a module (like related blogs)

This should make the document structure correct, but we need to check more pages when working on this.

Putting this on Elgg 6 as it will probably break a lot of HTML/JS/CSS
